src/coredata.cs

```diff
@@ -305,6 +342,10 @@ public interface NSFetchRequestExpression {
 		bool IsCountOnly { [Bind ("isCountOnlyRequest")] get;}
 	}
 
+	[BaseType (typeof (NSObject))]
```

Comment:
`[BaseType]` must **only** be used when you have a `[Model]` and this is **not** needed here.